main.go

```diff
@@ -92,6 +92,8 @@ type GlobalOptions struct {
 	FilterFiles         []string `short:"F" long:"filter-file" description:"Log file(s) to exclude from --file glob. May have multiple values, including multiple globs." yaml:"filter-file,omitempty"`
 	RenameFields        []string `long:"rename_field" description:"Format: 'before=after'. Rename field called 'before' from parsed lines to field name 'after' in Honeycomb events. May have multiple values." yaml:"rename_field,omitempty"`
 
+	LogLevel string `long:"log_level" description:"Set the log level. Valid values are 'debug', 'info', 'warn', 'error', 'fatal', 'panic'." default:"info" yaml:"log_level,omitempty"`
```

Comment:
Wow, I didn't know logrus supported `fatal` and `panic` levels. I wonder if users would find them useful when using honeytail

Reply:
Yeah, probably not. I let Co-Pilot auto generate the comment here 😄 

We have log entries for debug, info, error and fatal. We could remove warn and panic as they are unused.

Reply:
Coming back to this, I think the comment is valid - panic and warn log levels are valid, we just don't use them.